Dockerfile-dev

```diff
@@ -0,0 +1,55 @@
+# Stage 1: Build frontend assets.
+FROM node:12.6-alpine as frontend-build
+
+ENV NODE_ENV production
+COPY .babelrc gulpfile.js package.json yarn.lock ./
+COPY scripts/gulp ./scripts/gulp
+COPY lms/static ./lms/static
+RUN yarn install --frozen-lockfile
+
+RUN yarn build
+
+# Stage 2: Build the rest of the app using build output from Stage 1.
+FROM python:3.6.9-alpine3.10
+MAINTAINER Hypothes.is Project and contributors
+
+# Install system build and runtime dependencies.
+RUN apk add libpq collectd collectd-disk supervisor
+
+# Create the lms user, group, home directory and package directory.
+RUN addgroup -S lms \
+  && adduser -S -G lms -h /var/lib/lms lms
+WORKDIR /var/lib/lms
+
+# Copy minimal data to allow installation of dependencies.
+COPY requirements.txt ./
+
+# Install build deps, build, and then clean up.
+RUN apk add --virtual build-deps \
+    build-base \
+    postgresql-dev \
+    python3-dev \
+  && pip3 install --no-cache-dir -U pip \
+  && pip3 install --no-cache-dir -r requirements.txt \
+  && pip3 install --no-cache-dir pip-tools pyramid_debugtoolbar pyramid_ipython \
```

Comment:
This is the only difference between dev and prod. We could simplify this by just doing FROM our custom image and then install these after maybe but then the COPY .. would happen before the install so I think it's prob simpler and more efficient to just have 2 Dockerfiles.